src/OpenTelemetry/Trace/Configuration/TracerBuilder.cs

```diff
@@ -91,7 +91,7 @@ public TracerBuilder AddProcessorPipeline(Action<SpanProcessorPipelineBuilder> c
         /// </summary>
         /// <typeparam name="TCollector">Type of collector class.</typeparam>
         /// <param name="collectorFactory">Function that builds collector from <see cref="ITracer"/>.</param>
-        public TracerBuilder AddCollector<TCollector>(
+        public TracerBuilder AddAdapter<TCollector>(
```

Comment:
Same on comments, generic types, parameter names, xml docs, etc.
```suggestion
        public TracerBuilder AddAdapter<TAdapter>(
```